launcher/sdw_notify/Notify.py

```diff
@@ -0,0 +1,174 @@
+"""
+Utility library for warning the user that security updates have not been applied
+in some time.
+"""
+import fcntl
+import logging
+import os
+
+from datetime import datetime
+
+sdlog = logging.getLogger(__name__)
+
+# The directory where status files and logs are stored
+BASE_DIRECTORY = os.path.join(os.path.expanduser("~"), ".securedrop_launcher")
+
+# The file and format that contains the timestamp of the last successful update
+LAST_UPDATED_FILE = os.path.join(BASE_DIRECTORY, "sdw-last-updated")
+LAST_UPDATED_FORMAT = "%Y-%m-%d %H:%M:%S"
```

Comment:
nit: using the variable from Updater.py or storing in Utils.py could help reduce duplication here